docs/syntax/sequenceDiagram.md

```
@@ -409,6 +409,91 @@ end
     B->>C: Hello Charley, how are you?
 ```
 
+### Nested Grouping (v\<MERMAID_RELEASE_VERSION>+)
+
+Boxes can be nested inside other boxes to represent hierarchical groupings. Nesting can go as many levels deep as needed. Each box can have its own color and label, and inner boxes are visually drawn inside their parent.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)